src/main/java/HandleGetRequest.java

```diff
@@ -0,0 +1,74 @@
+package com.td.HttpServer;
+
+import java.util.*;
+import java.io.IOException;
+
+public class HandleGetRequest extends HttpHandleRequest {
```

Comment:
I guess I was thinking ahead. I was planning on making each type of request. i.e. GET , POST, PUT have its own corresponding class to handle that type of request. However there were some things that would be universal that are in the base class, such as the headers for the server and the date.